serverlessworkflow/modules/ROOT/pages/_common-content/productized-instructions.adoc

```diff
@@ -0,0 +1,34 @@
+:page-partial:
+
+:rhbq_config_registry_url: https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/quarkus-2-7/guide/f93c45bd-4feb-4f74-a70a-022e9fb41957#_9064727c-4b8a-4068-a9ba-8de6b258a14a
+:rhbq_config_maven_url: https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/quarkus-2-7/guide/f93c45bd-4feb-4f74-a70a-022e9fb41957#proc_download-maven_quarkus-getting-started
+
+[[configurint-maven-rhbq]]
+== Configuring your Maven project to Red Hat build of Quarkus and OpenShift Serverless Logic
+
+To use the Red Hat Build of Quarkus (RHBQ) libraries you have to configure your local Maven installation.
+
+.Procedure
+. Follow the instructions in the RHBQ documentation, sections: link:{rhbq_config_maven_url}[Configuring the Maven settings.xml file for the online repository] and link:{rhbq_config_registry_url}[Configuring Quarkus extension registry client].
++
+. Add the following excerpt to your project's `pom.xml` file:
```

Comment:
This is included before the Quarkus CLI/Maven plugin is executed, so either we split this in 2 parts where registry and Maven settings config will be included before the command and kogito-bom inclusion will be included after or here we can say to include this pom after the project is generated.

Reply:
I don't see it as a problem really. We may change the wording a bit, but splitting the procedure feels bad IMO. @hmanwani-rh what do you think?

Reply:
I think something as `After project is created, add the following excerpt ...`